openreview/tools.py

```diff
@@ -703,3 +698,39 @@ def post_submission_groups(client, conference_id, submission_invite, chairs):
             readers=[conference_id, chairs],
             signatories=[]))
 
+def get_submission_invitations(client, status='all'):
+    '''
+    Returns a list of invitations ids according to the status.
+    
+    :arg client: Object of :class:`~openreview.Client` class
+    :arg status: Status filter for invitations. Default value is "all". This is a string with following possible value: all, open and closed.
+    '''
+    
+    try:
+        # Get a list of all group with invitations satisfying the regex
+        groups = client.get_invitations(regex='.*/-/.*submission')
+        
+        # For each group in the list, append the group ids to a list
+        invitation_ids = []
+        
+        for g in groups :
+            
+            #Calculate the epoch for current timestamp
+            now = int(time.time()*1000)
+
+            # Assume the invitation is closed unless proven open
+            due = 0
+
+            if g.to_json()['duedate'] != None and now < g.to_json()['duedate']:
+                due = 1
+            
+            if status == "open" and due == 1:
```

Comment:
@mspector I need your opinion about how to filter the invitations.

Reply:
I think we don't need to filter the result locally, we can add the filter to the get_invitations call.

Reply:
do you think it should be filtered in the backend, or in the python library?

Reply:
the homepage of openreview gets the open submissions using these parameters:

```
regex: .*/-/[Ss]ubmission
minduedate: 1531505712680
```

I'm not sure if minduedate is implemented in the library.